file-formats/doma/type/zif_aff_doma_v1.abap

```diff
@@ -0,0 +1,112 @@
+INTERFACE zif_aff_doma_v1 PUBLIC.
+
+  "! <p class="shorttext">Plus - Minus sign for domains</p>
+  "! Plus - Minus sign for domains
+  TYPES ty_sign                 TYPE c.
+
+  "! <p class="shorttext">Case sensiive flag for domains</p>
+  "! Case sensiive flag for domains
+  TYPES ty_case_sensitive       TYPE c.
+
+  "! <p class="shorttext">Data type for the domain</p>
+  "! Data type for the domain
+  TYPES ty_data_type            TYPE c LENGTH 4.
+
+  "! <p class="shorttext">No Characters</p>
+  "! Number of Characters
+  TYPES ty_numb_char            TYPE c LENGTH 6.
+
+  "! <p class="shorttext">Decimals</p>
+  "! Decimals
+  TYPES ty_decimals             TYPE c LENGTH 6.
+
+  "! <p class="shorttext">Output Length</p>
+  "! Output Length
+  TYPES ty_output_length        TYPE c LENGTH 6.
```

Comment:
Aren't the fields "numbers of characters", "decimals" and "output length" numbers? If yes, I suggest to use `TYPE I`, here.

If we want to limit it to certain numbers we might want to use the annotations `$maximum` and/or `$minimum`

Reply:
From what I see in table D001L and in transaction SE11 they seem to be NUMC, however if you think TYPE I would be better its ok by me.

Reply:
For other object types we suggested to use integer with `$maximum` and `$minimum`. We should use the numeric only, if we want to keep the leading zeros.

Like this:

 ```suggestion
  "! $minimum 0
  "! $maximum 999999
  TYPES ty_numb_char            TYPE i.

  "! <p class="shorttext">Decimals</p>
  "! Decimals
  "! $minimum 0
  "! $maximum 999999
  TYPES ty_decimals             TYPE i.

  "! <p class="shorttext">Output Length</p>
  "! Output Length
  "! $minimum 0
  "! $maximum 999999
  TYPES ty_output_length        TYPE i.
```